app/org/sagebionetworks/bridge/stormpath/StormpathAccount.java

```diff
@@ -141,6 +148,14 @@ public void setConsentSignature(ConsentSignature signature) {
         encryptJSONTo(consentSignatureKey, signature);
     }
     @Override
+    public List<ConsentSignature> getConsentSignatureHistory() {
+        return decryptJSONFrom(consentSignatureHistoryKey, CONSENT_HISTORY_TYPE);
+    }
+    @Override
+    public void setConsentSignatureHistory(List<ConsentSignature> signatures) {
+        encryptJSONTo(consentSignatureHistoryKey, signatures);
+    }
```

Comment:
Can we add something to StormpathAccountTest, to make sure our code for encrypting/decrypting a list of consent signature histories works correct?